otx/algorithms/common/adapters/mmcv/models/backbones/torchvision_backbones.py

```diff
@@ -131,7 +131,7 @@ def multioutput_forward(self, x):
             print(temp_s)
         if i == last_stage:
             break
-    return outputs
+    return tuple(outputs)
```

Comment:
Just wondering: can we handle all forward function implemented on torchvision backbones by using above 3 functions? (resnet, shufflenet, multioutput forward)

Reply:
Most `torchvision backbones` use layers in `self.layers` or `self.features`. In such cases, it is patched to use `multioutput forward`, and resnet and shufflenet are configured slightly differently, so additional forward patching was required.